Platform.PCL/Realm.Sync.PCL/Permissions/PermissionChangePCL.cs

```diff
@@ -18,7 +18,7 @@
 
 using System;
 
-namespace Realms.Sync.Permissions
+namespace Realms.Sync
 {
     /// <summary>
     /// Objects of this class allow to change permissions of owned Realms.
```

Comment:
Bad grammar _allow to change permissions_ maybe rewrite as _are used to change permissions_.